sdk/node/test/accessTokens.js

```diff
@@ -64,6 +64,24 @@ describe('Access token', () => {
       .then(resp => expect(resp.items.map(item => item.id)).to.not.contain(tokenId))
   })
 
+  describe('queryAll', () => {
+    it('success example', () => {
+      let created
+      const queried = []
+
+      return createToken().then(token =>
+        created = token
+      ).then(() =>
+        client.accessTokens.queryAll({}, (token, next, done) => {
+          queried.push(token)
```

Comment:
Would it be a cleaner test to push `token.id` here, and then have the expectation be:

```
expect(queried).to.include(created.id)
```